### DISCUSSION TITLE
自定义规则不生效

### DISCUSSION THREAD

**Discussion by EricZhou05** [vernesong/OpenClash#4147] [Q&A]
捣鼓了半天，真的搞不懂哪里出了问题呜呜呜
添加的自定义规则无法生效，但确认系统已经添加到clash规则里面了，但openclash还是匹配了默认的规则。
已经确认自定义规则的行数在openclash“漏网之鱼”默认规则之上
![PixPin_2024-10-31_11-42-36](https://github.com/user-attachments/assets/36cce814-78de-46b0-8ba0-bebed578a36c)
![PixPin_2024-10-31_11-41-35](https://github.com/user-attachments/assets/7ed14155-019b-46a5-9bd3-10befe0dd952)
![PixPin_2024-10-31_11-42-05](https://github.com/user-attachments/assets/1f77e8f1-f18c-4c3d-9dea-f9c384cde632)
![PixPin_2024-10-31_11-51-45](https://github.com/user-attachments/assets/a3717cce-6070-4570-a19f-9d41bf7e1643)

**Comment by morytyann** [comment]
拼错了，是`paypal`，不是`payple`……

**Comment by EricZhou05** [comment]
NB
我真是个大聪明
